### PR TITLE
[IMP] l10n_nz: add tax report

### PR DESCRIPTION
**Pull request by oco-odoo**

### CODE REVIEW

**Comment by oco-odoo** [comment]
@jco-odoo Clear for merging. Maybe we should reduce the length of tag names, and just call them box 1, box 2, ... But we need to consider the usability of this.

File: addons/l10n_nz/data/account_tax_report_data.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        <field name="code">BOX5</field>
+        <field name="country_id" ref="base.nz"/>
+    </record>
+
+    <record id="tax_report_box6" model="account.tax.report.line">
+        <field name="name">[BOX 6] Zero-rated supplies in Box 5</field>
+        <field name="tag_name">BOX 6</field>
+        <field name="parent_id" ref="tax_report_sale_and_income"/>
+        <field name="sequence" eval="2"/>
+        <field name="code">BOX6</field>
```
Comment (jco-odoo) [inline addons/l10n_nz/data/account_tax_report_data.xml]: Is it possible to make this name more unique?  (e.g. NZBOX6, check other localizations...)

File: addons/l10n_nz/data/account_tax_report_data.xml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        <field name="parent_id" ref="tax_report_purchases_and_expenses"/>
+        <field name="sequence" eval="4"/>
+        <field name="formula">((BOX11 * 3)/23)</field>
+        <field name="country_id" ref="base.nz"/>
+    </record>
+
+    <record id="tax_report_box15" model="account.tax.report.line">
+        <field name="name">[BOX 15]  Difference between BOX10 and BOX14</field>
+        <field name="sequence" eval="3"/>
+        <field name="formula">((((BOX5 - BOX6) * 3)/23) + BOX9) - ((BOX11 * 3)/23)</field>
```
Comment (jco-odoo) [inline addons/l10n_nz/data/account_tax_report_data.xml]: It would be better to se the difference BOX10 - BOX14 (by also giving them codes)

**Comment by smetl** [comment]
@oco-odoo  @jco-odoo  What is the state of this PR?

**Comment by jco-odoo** [comment]
@rgo-odoo Can you just adapt the "code" field in order to have something more unique?

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**